HackIllinois/UI/TableView/Cells/HIBubbleCell/HIEventCell.swift

```diff
@@ -180,8 +180,7 @@ extension HIEventCell {
             sponsorLabel.leadingAnchor.constraint(equalTo: sponsorImageView.trailingAnchor, constant: eventCellSpacing + 1).isActive = true
             sponsorLabel.centerYAnchor.constraint(equalTo: sponsorImageView.centerYAnchor).isActive = true
         }
-        let locationLabel = HILabel(style: .newLocation)
-        locationLabel.text = "Online"
+        let locationLabel = HILabel(style: .newLocation); locationLabel.text = "Online"
```

Comment:
Can we pass in `text` as a parameter into `HiLabel()`?

Reply:
Yep! I will work on those HILabel changes asap.